backend/src/app.ts

```diff
@@ -102,3 +112,15 @@ function TournamentRegistrationsRepository() {
   }
   return new InMemoryTournamentRegistrationsRepository();
 }
+
+function PlayerProfilesRepository() {
+  //TODO add later above repositories ???
+  // if (process.env.MONGO_REPOSITORIES === 'ENABLED' && process.env.TOURNAMENTS_REGISTRATIONS_DATABASE === 'MONGO') {
+  //   return new MongoPlayerProfilesRepository();
+  // }
+  // if (process.env.POSTGRES_REPOSITORIES === 'ENABLED' && process.env.TOURNAMENTS_REGISTRATIONS_DATABASE === 'POSTGRES') {
+  //   return new PostgreSqlPlayerProfilesRepository();
+  // }
```

Comment:
dobrze by było nie mergować do developa zakomentowanego kodu, bo później gdzieś coś może przypadkiem zostać :P

Reply:
to może w sumie iść do developa, bo w kolejnym tasku będę dołączał MONGO repo do tego, więc i tak od razu to zniknie :-)